### ISSUE TITLE
Swagger for OData .Net Guide

### ISSUE THREAD

**Issue by ashinzekene** [OData/WebApi#2543]
Sorry is there any guide on enabling swagger for Odata WebAPI endpoints?

**Comment by ashinzekene** [comment]
Hi @Sreejithpin do you have any idea of how to go about this?

**Comment by marabooy** [comment]
@ashinzekene are you using 7.x or 8.x release?

**Comment by marabooy** [comment]
On 8.x [Check this guide](https://github.com/OData/AspNetCoreOData/tree/master/sample/ODataRoutingSample) On 7.x do checkout [this thread ](https://github.com/OData/WebApi/issues/2024)

**Comment by ashinzekene** [comment]
Thanks @marabooy, those are for `.NET Core` not `.Net Web API`

**Comment by marabooy** [comment]
@ashinzekene you can try the [following package](https://github.com/microsoft/OpenAPI.NET.OData) to convert the EdmModel to open-api descriptions that you can then serve using [Swashbuckle](https://github.com/domaindrivendev/Swashbuckle.WebApi) or any swagger-ui component.

**Comment by ashinzekene** [comment]
Your solution is also for ASP.Net Core @jacobmohl  🥲

**Comment by jacobmohl** [comment]
Yes sorry for the confusion. Something similar should be possible on .Net Framework as well.

**Comment by ashinzekene** [comment]
Yeah, trying to see if I can do that, the `IOperationFilter` signature is quite different for `.NET framework`

**Comment by ashinzekene** [comment]
Couldn't get this to work for .NET Framework
